src/main/java/org/jenkinsci/plugins/workinghours/model/ExcludedDate.java

```diff
@@ -41,63 +43,37 @@
  */
 public class ExcludedDate extends AbstractDescribableImpl<ExcludedDate> {
 
-    /**
-     * The display name of this rule.
-     */
-    private String name;
-
-    /**
-     * The date for this rule.
-     */
-    private String date;
+    private DataContainer dataContainer = null;
+    private String jsonData = "";
+    private static Gson gson = null;
 
     /**
-     * Constructs an ExcludedDate object.
-     *
-     * @param name Display name for date, e.g. "New Year's Day".
-     * @param date The date to exclude.
+     * Set or update the fields with a json which is then
```

Comment:
Nit, but can you finish this comment sometime before we merge this into the dev branch?